modules/microshift-install-multus-running-cluster.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-installing-on-running-cluster_{context}"]
+= Installing the Multus CNI plugin on an running cluster
+
+If you want to attach additional networks to a pod for high-performance network configurations, you can install the {microshift-short} Multus RPM package. After installation, a host restart is required to recreate all the pods with the Multus annotation.
+
+[IMPORTANT]
+====
+The Multus RPM should not be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
```

Comment:
If the customer needs `sudo`/`root` permissions, it is a good idea to call that out as prerequisite. This is inconsistently applied in reviews, but I think it is a good practice. That said, I know that there are some exceptions with MicroShift, so feel free to disregard if this comment is not relevant.

Reply:
That's a good callout. There are no `roles` in MicroShift, but the admin still needs root access to the host in this case. Shall add.